doc/api/https.md

```diff
@@ -23,6 +23,18 @@ added: v0.3.4
 This class is a subclass of `tls.Server` and emits events same as
 [`http.Server`][]. See [`http.Server`][] for more information.
 
+### server.close([callback])
+<!-- YAML
+added: v0.1.90
+-->
+- `callback` {Function}
+
+See [`http.close()`][] for details.
```

Comment:
`http.close()` is a little bit ambiguous. Maybe

```
See [`server.close()`][`http.close()`] from the HTTP module for details.
```

Reply:
I only moved the description of http.close() to another location. But it would be a nice change to make the description better.